### ISSUE TITLE
XML path evaluation - need to get the type of the element first before text()

### ISSUE THREAD

**Issue by seymen** [apickli/apickli#16]
If the element resolved from the xpath is an attribute, text() method fails.

Need to do type() first and depending on the type, do text() or value()

https://github.com/polotek/libxmljs/wiki/Node
https://github.com/polotek/libxmljs/wiki/Attribute

**Comment by seymen** [comment]
```js
		case 'xml':
			var xml = libxmljs.parseXml(content);
			var node = xml.get(path);
			if (node.type() === 'attribute') {
				return node.value();
			} else { // 'comment' or 'element'
				return node.text();
			}

```

**Comment by seymen** [comment]
fixed by ea85f5746f98f2d13bab3829ba47d6e48de463b2
